api/octokit.ts

```diff
@@ -393,41 +393,6 @@ export class OctoKit implements GitHub {
 		})
 	}
 
-	protected async getItemIdFromIssueProjectNext(
```

Comment:
Hello 👋🏾 actually this code is needed, to remove an issue from GitHub from new projects we need to get the `ITEM_ID` which is assigned at the time we add an issue to a project, so far there is no way through the API (beta ) to get that. The solution I found was:

- Perform the action of adding the issue to the project (if the issue is already there, it will return the `ITEM_ID`)
- Take that `ITEM_ID`
- And pass it to [deleteProjectNextItem mutation](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/using-the-api-to-manage-projects )


You can take a look at [Github Action ](https://github.com/grafana/grafana/runs/5480557600?check_suite_focus=true#step:4:3597), when I was removing the label, there is a difference between ` issueNodeId` and the `projectNextItem id` project

```
Running getItemIdFromIssueProjectNext with: projectNodeId: PN_kwDOAG3Mbc2Pyw issueNodeId: I_kwDOAOaWjc4_VDUk
getItemIdFromIssueProjectNext results{"addProjectNextItem":{"projectNextItem":{"id":"PNI_lADOAG3Mbc2Py84AK_qq"}}}
```

Reply:
Hello @axelavargas 👋,

I did not know that Github also has a projectNextNodeId 🤦  and not just a `nodeId` which is already part of the event the github actions gets as an Input. 

I have switched the logic up a bit on how to get the right ID, see https://github.com/grafana/grafana-github-actions/pull/83/commits/4e50a024c2f71af6c94eda797394901a21e0ef47, https://github.com/grafana/grafana-github-actions/pull/83/commits/61b32dd97c15d5bfa0647c6363902dc7414d3df6

And also set up [commands](https://github.com/grafana/github-actions-testrepo/blob/main/.github/issue-commands.json#L11-L26) in our test repo to [use this branch](https://github.com/grafana/github-actions-testrepo/blob/main/.github/workflows/issue-commands.yml#L20):
- https://github.com/grafana/github-actions-testrepo/issues/67 (add / remove label `plugins`)
- You can see the logs under https://github.com/grafana/github-actions-testrepo/actions

So far as I can see it seems to be working, could you please validate?